vmdb/app/helpers/container_service_helper/textual_summary.rb

```diff
@@ -53,12 +52,16 @@ def textual_portal_ip
     {:label => "Portal IP", :value => @record.portal_ip}
   end
 
-  def textual_protocol
-    {:label => "Protocol", :value => @record.protocol}
-  end
 
-  def textual_container_port
-    {:label => "Container Port", :value => @record.container_port}
+  def textual_port_config(single_conf)
+    name = single_conf.name
+
+    name = "Unnamed" if name.nil? || name.empty?
+
+    {
+      :label => name,
+      :value => "#{single_conf.protocol} port:#{single_conf.port} to port:#{single_conf.target_port}"
```

Comment:
```go
// Optional: The target port on pods selected by this service.
// If this is a string, it will be looked up as a named port in the
// target Pod's container ports. If this is not specified, the value
// of Port is used (an identity map).
 TargetPort util.IntOrString `json:"targetPort" description:"the port to access on the pods targeted by the service; defaults to the service port"
```
How about this:
![port_on_pod](https://cloud.githubusercontent.com/assets/11256940/7180521/05ab78ca-e44b-11e4-9259-a3b999f14d12.png)